code/obj/item/bowling.dm

```diff
@@ -72,6 +85,18 @@
 							src.hitWeak(hitMob, user)
 		return
 
+	proc/pre_attack_damage_modifier(obj/item/parent_item, atom/A, mob/user)
+		var/mob/living/carbon/human/human_user = user
+		if(human_user.w_uniform && istype(human_user.w_uniform, /obj/item/clothing/under/gimmick/bowling))
+			//bashing someones skull in with a bowling ball should hurt if you are worthy of the bowling ball
+			src.force = 20
```

Comment:
```suggestion
			src.force = 15
```
20 is a LOT, that's the same as the fireaxe and reverse katana. Since this gives you four of these and they can't be used if stolen I think 20 is too far.